modules/sdk/gradle-plugins-maven-plugin-builder/src/main/java/com/liferay/gradle/plugins/maven/plugin/builder/BuildPluginDescriptorTask.java

```diff
@@ -78,6 +78,7 @@ public BuildPluginDescriptorTask() {
 			Conf2ScopeMappingContainer.COMPILE);
 		_configurationScopeMappings.put(
 			"provided", Conf2ScopeMappingContainer.PROVIDED);
+		_pomRepositories.put("provided", Conf2ScopeMappingContainer.PROVIDED);
```

Comment:
Here you should add the default `liferay-public` - `http://cdn.repository.liferay.com/blah/blah` (copy from `liferay-plugins/ivy-settings.xml`).